docs/DeveloperGuide.md

```diff
@@ -128,10 +126,10 @@ the filtered transaction with a success message.
 - 2a. User enters an incorrect format.
     - 2a1. UniCa$h displays an error message, requests for correct format.
     - 2a2. User enters command with new format.
-  
-    Steps 2a1-2a2 are repeated until the format entered is correct.
 
-    Use case resumes from Step 3.
+  Steps 2a1-2a2 are repeated until the format entered is correct.
+
+  Use case resumes from Step 3.
 
```

Comment:
Not entirely sure what's the change here so I can't revert this

Reply:
<img width="972" alt="image" src="https://github.com/AY2324S1-CS2103-T16-3/tp/assets/19762596/2e49e82f-1e54-4d28-8bae-9558e4900d92">

`Steps` and `Use case` were initially aligned with `- 2a1` maybe you might wanna try adding a space or a few, I think this was probably caused by IntelliJ also